default/en/source_040.md

```diff
@@ -4,4 +4,10 @@ The field **Cataloging source** indicates the organization that created the orig
 
 ### Language of cataloging (040 $b)
 
-Indicate your cataloging language. Select from English, French, German, Italian, Portuguese, or Spanish. 
+Indicate your cataloging language. Select from the following.
+- English  
+- French  
+- German  
+- Italian  
+- Portuguese  
+- Spanish   
```

Comment:
Polish?

Reply:
No, in fact it is not an official cataloging language.

Reply:
That's a shame, because it's the only language on this list that is supported by Muscat, but not an official cataloguing language.